gum/backend-libdwarf/gumsymbolutil-libdwarf.c

```diff
@@ -551,6 +644,9 @@ gum_collect_module_functions (const GumModuleDetails * details,
   gum_elf_module_enumerate_symbols (entry->module,
       gum_collect_symbol_if_function, NULL);
 
+  gum_elf_module_enumerate_symbols (entry->module,
+      gum_collect_symbol, NULL);
```

Comment:
How about adding this logic to `gum_collect_symbol_if_function` instead? (To avoid walking the symbols twice. The predicate is the same as well.)